docker/build-tools/Dockerfile

```
@@ -628,10 +628,13 @@ ENV CARGO_HOME=/home/.cargo
 ENV RUSTUP_HOME=/home/.rustup
 # hadolint ignore=DL4006
 RUN curl --proto '=https' -v --tlsv1.2 -sSf https://sh.rustup.rs | \
-    sh -s -- -y -v --default-toolchain ${RUST_VERSION} --profile minimal --component rustfmt clippy &&\
-    /home/.cargo/bin/rustup default ${RUST_VERSION} &&\
+    sh -s -- -y -v --default-toolchain ${RUST_VERSION} --profile minimal && \
+    /home/.cargo/bin/rustup default ${RUST_VERSION} && \
+    /home/.cargo/bin/rustup component add rustfmt clippy llvm-tools && \
+    /home/.cargo/bin/cargo install rustfilt && \
     mv /home/.cargo/bin/* /usr/bin
 
+RUN cargo install rustfilt
```

These two lines look duplicative? And I think not actually needed for this PR, we're just trying to update Go here, not add this demangler for code coverage reporting like in https://github.com/istio/tools/pull/3022

```suggestion
    mv /home/.cargo/bin/* /usr/bin

```

---

removed it. Thanks!!